### ISSUE TITLE
Fix S2183 FN: Support unsigned right-shift operator (>>>)

### ISSUE THREAD

**Issue by gregory-paidis-sonarsource** [SonarSource/sonar-dotnet#6294]
### Description

FN for S2183 (DoNotShiftByZeroOrIntSize).

Using C#11 Unsigned right-shift operator (>>>), the rule is not raising an issue.

### Repro steps

```csharp
        int i;
        ulong j = 0;

        i = i >>> 60; // FN
        i >>>= 40; // FN

        q = q >>> 64; // FN
        q >>>= 70; // FN
```

### Expected behavior

The rule should raise an issue.

### Actual behavior

The rule does not raise an issue.
